acceptance-tests/support/steps/input-steps.ts

```diff
@@ -62,3 +62,9 @@ Then("the value of the text field {} should be {string}", async function (this:
     );
     assert.equal(elementAttributeValue, attributeValueToCheck);
 });
+
```

Comment:
To avoid linter errors insert the following line here : 
```
// eslint-disable-next-line
```
In this case disabling linter makes sense, because we need the parameter for the function to work as intended (to insert the condition in the step definition), but we do not need to use it inside the function.

Reply:
> // eslint-disable-next-line

Boris, are you sure this will resolve the ESLINT Issue which was that Condition is being declared but not used, hence my Console.log?

Reply:
As you can see, there are no lint issues in the latest build.